### PR TITLE
[DOC, MRG] weight & height keys of subject_info dict is documented

### LINKED ISSUE #10964: Subject weight & height info keys are not documented
We support `info['subject_info']['weight']`:
https://github.com/mne-tools/mne-python/blob/c12a02000776adb9df18da55ea60c50cbf83827d/mne/io/meas_info.py#L1694

but the `weight` key is not documented at:
https://mne.tools/dev/generated/mne.Info.html

### PR DESCRIPTION
**Pull request by sena-neuro**
Closes #10964.

`height` and `weight` keys of  `subject_info` entry in `mne.Info` are documented.

### CODE REVIEW

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -24,6 +24,7 @@ Current (1.2.dev0)
 Enhancements
 ~~~~~~~~~~~~
 - EEGLAB files (saved as MAT versions less than v7.3) can now be imported with :func:`mne.io.read_raw_eeglab` without the optional dependency ``pymatreader`` (:gh:`11006` by `Clemens Brunner`_)
+- Document ``height`` and ``weight`` keys of  ``subject_info`` entry in :class:`mne.Info` (:gh:`10964` by :newcontrib:`Sena Er`)
```
Comment (drammock) [inline doc/changes/latest.inc:27]:
```suggestion
- Document ``height`` and ``weight`` keys of  ``subject_info`` entry in :class:`mne.Info` (:gh:`11019` by :newcontrib:`Sena Er`)
```

FYI in the changelog we list the PR number where the problem was fixed, not the issue number where the problem was reported.

Also, this entry isn't really an "enhancement" (which means "new functionality").  The closest fit for missing documentation would be the `Bugs` section. Can you move it, and correct the PR number?

**Review by drammock — CHANGES_REQUESTED** [review]
Thanks for this contribution! See below for fixes needed on the changelog entry.

File: doc/changes/latest.inc (hunk 1 above)
Comment (sena-neuro) [reply doc/changes/latest.inc:27]: Yes, of course. Thanks!

**Review by agramfort — APPROVED** [review]

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
only CI failure is a tolerance fail in `test_make_forward_dipole` ([here](https://github.com/mne-tools/mne-python/runs/7765566654?check_suite_focus=true#step:14:3636), unrelated to this PR).

**Pull request merged**